hack/k8s/example-ro.template.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: datamon-ro-demo
+spec:
+  selector:
+    matchLabels:
+      app: datamon-ro-demo
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: datamon-ro-demo
+    spec:
+      containers:
+      - name: demo-shell
+        image: fuse-demo-shell:latest
+        imagePullPolicy: "Never"
+        command: ["/bin/tini"]
+        args: ["--", "/bin/bash"]
+        stdin: true
+        tty: true
+        volumeMounts:
+        - mountPath: /tmp/mount
+          name: fuse-mountpoint
+          mountPropagation: "HostToContainer"
+      - name: datamon-sidecar
+        image: fuse-demo-sidecar:latest
+        imagePullPolicy: "Never"
+        command: ["datamon"]
+        args: ["bundle", "mount", "--bundle", "1Kbtk5Mo0M8ck2lcnuhMNYiV9DO", "--repo", "ransom-datamon-test-repo", "--destination", "/tmp", "--mount", "/tmp/mount"]
+        securityContext:
+          privileged: true
+        stdin: true
+        tty: true
+        volumeMounts:
+        - mountPath: /tmp/mount
+          name: fuse-mountpoint
+          mountPropagation: "Bidirectional"
```

Comment:
Do we need bidirectional for within pod sharing?

Reply:
yes, i have tried toggling this to [other strings](https://kubernetes.io/docs/concepts/storage/volumes/#mount-propagation) and re-running the `fuse-demo/` scripts to drop into the `demo-shell`, and `ls /tmp/mount` from the demo shell only displays the demo files with bidirectional mount propagation.

in particular, what was not obvious to me (at the level of k8s abstraction) is why there's host->container unidirectional propagation and not container->host.

Reply:
I guess the mounts need to go back to the os for it to share with the other container cause pod is a k8s abstraction and as far as the OS is concerned these are separate containers.